t04_refactored.py

```diff
@@ -146,9 +146,44 @@ def team_5_adv():
 
 ###################################################################################
 
+
 def team_6_adv():
-    pass
-    # TODO Add your code here
+    path = (input
+            ("Which path will you take? [Left/Middle/Right]"))
+
+    second_chance = (input
```

Comment:
Shouldn't they get a chance before a second chance (ie., nothing is done after the first input)